firefox-plugins-installer

```diff
@@ -81,11 +77,7 @@ fetch_flash_metadata() {
 
     if [ -z ${FLASH_LKGV_URL} ]; then
         echo "Failed to retrieve the last known good version of the flash plugin"
```

Comment:
Not your bug, but: s/flash/Flash/?

Reply:
Maybe also worthwhile ensuring that this returns a different exit status from other errors, so it can be detected if needed, by a calling script?

Reply:
(Having never looked at this code before:) Does something retry the Flash plugin download if it fails?

Reply:
(Yes: `firefox-plugins-installer` is run in the background every time the user starts Firefox from its `.desktop` file)

Reply:
> Maybe also worthwhile ensuring that this returns a different exit status from other errors, so it can be detected if needed, by a calling script?

For better or for worse, this script always exits with success, so there's no real gain to changing the return code for this function.